extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -332,19 +332,25 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	public get tableOfContentPaths() {
 		return this._tableOfContentPaths;
 	}
+
 	getUntitledNotebookUri(resource: string): vscode.Uri {
-		let title = this.findNextUntitledFileName(resource);
-		let untitledFileName: vscode.Uri = vscode.Uri.parse(`untitled:${title}`);
-		if (!this._allNotebooks.get(untitledFileName.fsPath)) {
+		let untitledFileName: vscode.Uri;
+		if (process.platform.indexOf('win') > -1) {
+			let title = path.join(path.dirname(resource), this.findNextUntitledFileName(resource));
```

Comment:
Why do it like this here? We've always just done process.platform === 'win32' - that's the only value returned for windows machines.

Reply:
ohh, I didn't know that. I'll make that change. Thanks!